test/check-storage-mountpoints

```diff
@@ -361,6 +410,18 @@ class TestStorageMountPoints(VirtInstallMachineCase, StorageCase):
 
     @nondestructive
     def testEncryptedUnlockCockpit(self):
+        """
+        Description:
+            Test 'Mount point mapping' with encrypted partitions created using Cockpit Storage
+
+        Expected results:
+            - The user can create an encrypted partition with cockpit storage
+            - The 'Unlock' dialog is displayed before entering the mount point mapping
```

Comment:
I'd rather say
`- The 'Unlock' dialog is available for unlocking the partitions before entering the mount point mapping`